### ISSUE TITLE
comment.lua:65: attempt to call field 'get_lines' (a nil value)

### ISSUE THREAD

**Issue by aravind2612krishna** [harrisoncramer/gitlab.nvim#216]
## Prerequsities

- [X] I'm on the latest version of the plugin
- [X] I've installed the required dependencies
- [ ] I've run `:h gitlab.nvim.troubleshooting` and followed the steps there

## Setup Configuration and Environment

Only difference from default I think is : 
```
      diffview = {
        imply_local = true,
      },
```

## Bug Description

A function seems to be missing in the reviewer/init.lua (`get_lines`)
``` 
E5108: Error executing lua: ...are/nvim/lazy/gitlab.nvim/lua/gitlab/actions/comment.lua:65: attempt to call field 'get_lines' (a nil value)
stack traceback:
        ...are/nvim/lazy/gitlab.nvim/lua/gitlab/actions/comment.lua:65: in function 'cb'
        .../.local/share/nvim/lazy/gitlab.nvim/lua/gitlab/async.lua:26: in function 'fetch'
        .../.local/share/nvim/lazy/gitlab.nvim/lua/gitlab/async.lua:34: in function 'fetch'
        .../.local/share/nvim/lazy/gitlab.nvim/lua/gitlab/async.lua:34: in function 'fetch'
        .../.local/share/nvim/lazy/gitlab.nvim/lua/gitlab/async.lua:60: in function <.../.local/share/nvim/lazy/gitlab.nvim/lua/gitlab/async.lua:47>
```

## Reproduction Steps

1. Check out the feature branch
2. Open Neovim
3. Create multi-line comment

**Comment by harrisoncramer** [comment]
Thank you, this will be fixed in the next release, probably in the next couple of days. If you want this fixed immediately you can use the `develop` branch of the plugin which now has the fix.

**Comment by harrisoncramer** [comment]
Fixed by #227, fixed in v2.0.4, closing.
